game-headed/src/main/java/org/triplea/game/client/ui/javafx/MainMenuPane.java

```diff
@@ -101,12 +106,16 @@ private void showLastMenu() {
 
   @FXML
   private void showHelp() {
-    triplea.open(UrlConstants.GITHUB_HELP);
+    open(UrlConstants.GITHUB_HELP);
   }
 
   @FXML
   private void showRuleBook() {
-    triplea.open(UrlConstants.RULE_BOOK);
+    open(UrlConstants.RULE_BOOK);
+  }
+
+  private void open(final String url) {
+    OpenFileUtility.openUrl(url, () -> log.warning("Desktop API not supported. Could not open " + url));
```

Comment:
side-note: ideally we would do this as a startup check, if not available then we'd hide the UI control and not provide it as an option.

Reply:
Good point, a simple disabled button + tooltip is much better than throwing an error.
Will do that in a follow-up PR.